index.html

```diff
@@ -30,16 +30,16 @@ <h1>Marco Antonio Hincapié Montes</h1>
       <button type="button" class="headline-button">Start collaboration</button>
```

Comment:
Hey @AntonioHincapie 😃 
You've done a tremendous job in your portfolio project specifically in the `accessibility` activity.  However, I ran your project link on aditus.io and it's raising an error on hover start in the Start Collaboration button. You may check the result here: https://www.aditus.io/button-contrast-checker/antoniohincapie-github-io-2022-04-05-at-23-30-22-963?failing